### PR TITLE
Add response headers setting

### PR DESCRIPTION
**Pull request by p3nt4**
I find it useful being able to modify response headers on the server level. 
I tried an implementation of it,.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var Server = require('./lib/server')
 module.exports = function (config) {
   var server = new Server(config);
   upgrade(server);
+  server.options.responseHeaders=config.responseHeaders;
```
Comment (houmark) [inline index.js]: Maybe add a space on each side of the `=`?

File: lib/server.js [hunk 2]
```diff
@@ -102,6 +102,13 @@ Server.prototype.handleRequest = function handleRequest (req, res) {
 
   // add utilites to req and res
   setupReqRes(server.options, req, res, function(err, next) {
+    if(server.options.hasOwnProperty("responseHeaders")){
```
Comment (houmark) [inline lib/server.js]: Missing space before last `{`.

**Comment by houmark** [comment]
Can you provide a use case on how to add headers so I can test it out when I have time?

**Comment by p3nt4** [comment]
`// staging.js
var deployd = require('deployd');

var server = deployd({
  port: 8081,
  env: 'dev',
  db: {
    host: '127.0.0.1',
    port: 27017,
    name: '',
    responseHeaders:{
	 Pragma: 'no-cache'
     },
    credentials: {
      username: '',
      password: ''
    }
  }
});

server.listen();

server.on('listening', function() {
  console.log("Server is listening");
});

server.on('error', function(err) {
  console.error(err);
  process.nextTick(function() { // Give the server a chance to return an error
    process.exit();
  });
});
`

File: lib/server.js [hunk 3]
```diff
@@ -102,6 +102,13 @@ Server.prototype.handleRequest = function handleRequest (req, res) {
 
   // add utilites to req and res
   setupReqRes(server.options, req, res, function(err, next) {
+    if(server.options.hasOwnProperty("responseHeaders")) {
+	for (var header in server.options.responseHeaders) {
+    	     if (server.options.responseHeaders.hasOwnProperty(header)) {
+        	res.setHeader(header, server.options.responseHeaders.header);
```
Comment (andreialecu) [inline lib/server.js]: Shouldn't this be `res.setHeader(header, server.options.responseHeaders[header]);` ?

**Comment by andreialecu** [comment]
This is a bit specific. 

You could achieve the same outcome by using a plugin like [dpd-router-middleware](https://github.com/ezra-obiwale/dpd-router-middleware) and using `ctx.res.setHeader(...);` from the event script.

Going to close this in favor of the above approach.

**Pull request closed without merging**